set-environment

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Could this file go in the `snap` directory too, or not?

Reply:
I *think* it can go in `snap/local`. I'll give it a try.

Reply:
It can. Done!